### PR TITLE
Updates from rspec-dev (2020-12-25)

### PR DESCRIPTION
**Pull request by JonRowe**
These are some updates, generated from rspec-dev's rake tasks.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,47 +24,63 @@ If you need help getting started, check out the [DEVELOPMENT](DEVELOPMENT.md) fi
...
-   `bundle exec rake generate:app generate:stuff` is executed.
-3. The example app is copied in to `tmp/aruba`
-4. Everything in `tmp/aruba/spec/*` is deleted apart from `spec/spec_helper.rb` and
-   `spec/rails_helper.rb`
-5. the cucumber suite executes, creating files in that app and executing them
-
-The best way to debug the app is to run a failing cucumber feature, which will
-leave the test files intact in `tmp/aruba`, then you can cd in to that director
-and run it in the bundle context of the aruba app.
+## Working on multiple RSpec gems at the same time
```
Comment (pirj) [inline CONTRIBUTING.md:46]: This doesn't apply here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -24,47 +24,63 @@ If you need help getting started, check out the [DEVELOPMENT](DEVELOPMENT.md) fi
...
-These issues are ones that have been labelled by the maintainers that we
-believe do not currently have enough information to be reproduced the RSpec
-team. While not directly counted by the GitHub contribution graph, we consider
-helping us to reproduce the issue with a repro case as an extremely meaningful
-contribution.
-
-### `Has reproduction case` issues
-
-These issues are the ones that have reproduction cases, able to start working on
-immediately. These are good ones to tackle to help us actively fix bugs.
```
Comment (pirj) [inline CONTRIBUTING.md:46]: I suggest keeping this.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -11,8 +11,8 @@ We welcome contributions from *everyone*. While contributing, please follow the
 If you'd like to help make RSpec better, here are some ways you can contribute:
 
   - by running RSpec HEAD to help us catch bugs before new releases
-  - by [reporting bugs you encounter](https://github.com/rspec/rspec-rails/issues/new?template=bug_report.md)
-  - by [suggesting new features](https://github.com/rspec/rspec-rails/issues/new?template=feature_request.md)
```
Comment (pirj) [inline CONTRIBUTING.md:15]: This is rather useful.

File: DEVELOPMENT.md [hunk 4]
```diff
@@ -129,31 +129,3 @@ $ bin/yard server --reload
...
 
 Then navigate to `localhost:8808` to view the rendered docs.
-
-### Publishing the documentation
-
-In the other documented rspec gems, we use a rake command in rspec-dev
-to generate documentation for [rspec.info](https://rspec.info/).
-As `rspec-rails` is no longer sync with the other gems in terms of versioning
-since `rspec-rails` 4, if you want to publish updated documentation you will
-need to run the rake task from this repository.
```
Comment (pirj) [inline DEVELOPMENT.md:139]: This is important.

**Review by pirj — CHANGES_REQUESTED** [review]
I guess some of the plaintext files don't apply here as well.

**Comment by JonRowe** [comment]
@pirj this contained some undesired changes which I reverted manually, script might need some tweaking to ignore some files.

**Review by pirj — APPROVED** [review]
Thanks!

**Pull request merged**